[2025-08-26T12:14:03.053+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-26T12:14:03.063+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generated_py_do_something.my_function manual__2025-08-26T12:14:02.271810+00:00 [queued]>
[2025-08-26T12:14:03.068+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generated_py_do_something.my_function manual__2025-08-26T12:14:02.271810+00:00 [queued]>
[2025-08-26T12:14:03.068+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-26T12:14:03.074+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): my_function> on 2025-08-26 12:14:02.271810+00:00
[2025-08-26T12:14:03.076+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-26T12:14:03.077+0000] {standard_task_runner.py:63} INFO - Started process 82 to run task
[2025-08-26T12:14:03.077+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'generated_py_do_something', 'my_function', 'manual__2025-08-26T12:14:02.271810+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5acebq5q']
[2025-08-26T12:14:03.077+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask my_function
[2025-08-26T12:14:03.093+0000] {task_command.py:426} INFO - Running <TaskInstance: generated_py_do_something.my_function manual__2025-08-26T12:14:02.271810+00:00 [running]> on host 1ed929767bb5
[2025-08-26T12:14:03.121+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generated_py_do_something' AIRFLOW_CTX_TASK_ID='my_function' AIRFLOW_CTX_EXECUTION_DATE='2025-08-26T12:14:02.271810+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-26T12:14:02.271810+00:00'
[2025-08-26T12:14:03.121+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-26T12:14:03.127+0000] {logging_mixin.py:188} INFO - Doing something inside AirFlow!
[2025-08-26T12:14:03.127+0000] {logging_mixin.py:188} INFO - Step: 0
[2025-08-26T12:14:03.127+0000] {logging_mixin.py:188} INFO - No data!
[2025-08-26T12:14:03.127+0000] {logging_mixin.py:188} INFO - Step: 2
[2025-08-26T12:14:03.127+0000] {logging_mixin.py:188} INFO - No data!
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - Step: 4
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - No data!
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - Step: 6
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - No data!
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - Step: 8
[2025-08-26T12:14:03.128+0000] {logging_mixin.py:188} INFO - No data!
[2025-08-26T12:14:03.128+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-26T12:14:03.128+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-26T12:14:03.131+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=generated_py_do_something, task_id=my_function, run_id=manual__2025-08-26T12:14:02.271810+00:00, execution_date=20250826T121402, start_date=20250826T121403, end_date=20250826T121403
[2025-08-26T12:14:03.178+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-26T12:14:03.189+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
